woocommerce/woocommerce
# Use admin password reset link on admin login screen

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/50195 .

This PR keeps users in the admin UI flow when they choose to reset password from the admin login screen. This is thought to be a better experience than sending admins to the front-end My Account page to reset their password.


It also solves a bug related to coming soon mode, because the My Account page is not accessible to logged-out users, which could prevent admin users from resetting their password.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Settings > Site visibility 
2. Set the store to Live and save
3. Logout and go to the admin login screen `/wp-login.php`
4. Click "Lost your password?"
5. See that the lost password screen is presented with the admin UI.
![image](https://github.com/user-attachments/assets/74a06d46-c399-4876-b957-1ba33479711d)

6. Go to the front-end of the store
7. Click "My account" in the navigation
8. Click "Lost your password?"
9. See that the lost password screen is presented with the front-end UI.

<img width="1792" alt="front end forget" src="https://github.com/user-attachments/assets/bfcd96fd-0007-4cb8-a050-3210f6a96bd2">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
